.template/addons/github/template.rb

```diff
@@ -1,3 +1,37 @@
 use_source_path __dir__
 
 directory '.github'
+
+# Split README.md file to multiple wiki pages
+original_readme = File.read 'README.md'
+
+create_file '.github/wiki/Getting-Started.md' do
+  get_content_between(original_readme, '## Getting Started', '## Testing')
+end
+
+create_file '.github/wiki/Testing.md' do
+  get_content_between(original_readme, '## Testing', '## CI/CD')
+end
+
+FileUtils.mv '.github/workflows/README.md', '.github/wiki/CI-CD.md'
+
+gsub_file 'README.md', /## Getting Started.*/m do
+  <<~README
+    ## Documentation
+
+    Please check out full documentation on the [wiki](link to github wiki goes here).
```

Comment:
Given that the Readme file is at the root level, we can define the link to wiki like this, [source](https://stackoverflow.com/a/28470884/2189968)


```suggestion
    Please check out full documentation on the [wiki](../../wiki).
```

Reply:
Nice~ 👍 , Updated in [7666a76](https://github.com/nimblehq/rails-templates/pull/317/commits/7666a761bf57776824557218ea71efde42608e22) 🎉